src/modules/trade/helpers/generate-trade.js

```diff
@@ -23,14 +23,17 @@ export const generateTrade = memoize((market, outcome, outcomeTradeInProgress, o
 
   const side = (outcomeTradeInProgress && outcomeTradeInProgress.side) || TRANSACTIONS_TYPES.BUY
   const numShares = (outcomeTradeInProgress && outcomeTradeInProgress.numShares) || null
+  const sharesFilled = (outcomeTradeInProgress && outcomeTradeInProgress.sharesFilled) || null
   const limitPrice = (outcomeTradeInProgress && outcomeTradeInProgress.limitPrice) || null
   const totalFee = (outcomeTradeInProgress && outcomeTradeInProgress.totalFee) || 0
+  const feePercent = (outcomeTradeInProgress && outcomeTradeInProgress.feePercent) || 0
   const gasFeesRealEth = (outcomeTradeInProgress && outcomeTradeInProgress.gasFeesRealEth) || 0
   const totalCost = (outcomeTradeInProgress && outcomeTradeInProgress.totalCost) || 0
-  const marketType = (market && market.type) || null
-  const minPrice = (market && market.minPrice) || null
+  const marketType = (market && market.marketType) || null
+  const minPrice = (market && typeof market.minPrice === 'number') ? market.minPrice : null
```

Comment:
I believe `typeof market.minPrice` should be a string, not a number.  (It may _currently_ be a number due to an outstanding augur-node bug, but it is supposed to be a string -- I suggest future-proofing this by allowing it to handle either type.)